### ISSUE TITLE
Manuscript instance segmentation

### ISSUE THREAD

**Issue by SB2020-eye** [SB2020-eye/Issues#1]
starting conversation...

**Comment by ekesdf** [comment]
I am gonna start with your 3 questions, you asked.

**Question: Just to make sure: you both used PixelLib, correct?**
Answer: Yes, we used PixelLib

**Question: Also, were the steps pretty straightforward?**
Answer: Yea, the part that takes the most effort is actually the labeling of your images. 
              The actual training is a piece of cake

**Question: Will I be able to do this without GPU?**
Answer: well running the model not but training for sure

**Comment by SB2020-eye** [comment]
Great. Thanks so much, @ekesdf 

My 14-year-old son has a computer with GPU he may let me use (lol) when necessary (and during some time he's not gaming).

So what should I do next, do you think? Try my hand at labeling, using Labelme (as recommended/required by PixelLib)?

(And if yes: I have a few images partially labeled (just boxes) using labelimg. Is there any way to transfer/utilize those? Or, I wonder, is it even worth it, since it's just boxes and Labelme will require segmentation anyways?)

**Comment by ekesdf** [comment]
So for the GPU of your son, I am pretty sure it will not work for the thing we want it to use.
There are 2 solutions for that. The first is I can create a so-called notebook that you can then run in your web browser using a service called Google Cola.(You basically get a PC with every thing installed and with a relatively powerful GPU for free). The second option is that I can train the model on my PC bc I have basically 2 GPUs that are powerful enough to train the model also for free

if those pre-labeld images are useful depends on your need: If you need the single char with the background, then yea 
otherwise, they are useless and there is no efficient way to transfer them 

and yea, Labelme is the right tool for this

**Comment by SB2020-eye** [comment]
Thank you so much. I'll start trying out Labelme. I'll be in touch.

**Comment by SB2020-eye** [comment]
@ekesdf , do i need to set up my data folders a specific way before using labelme?

**Comment by ekesdf** [comment]
> @ekesdf, do I need to set up my data folders a specific way before using labels?

No, you don't need a specific folder structure just put them all in the same folder and that's it

If you have further questions ask me whenever you want
